app/views/foods/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: food) do |form| %>
+  <% if food.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(food.errors.count, "error") %> prohibited this food from being saved:</h2>
+
+      <ul>
+        <% food.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :measurement_unit, style: "display: block" %>
+    <%= form.text_field :measurement_unit %>
+  </div>
+
+  <div>
+    <%= form.label :price, style: "display: block" %>
+    <%= form.text_field :price %>
+  </div>
+
+  <div>
+    <%= form.label :quantity, style: "display: block" %>
+    <%= form.number_field :quantity %>
+  </div>
+
+  <div>
+    <%= form.label :user_id, style: "display: block" %>
+    <%= form.text_field :user_id %>
```

Comment:
- [ ] We want to ensure privacy, whereby users can create and delete their own food list. Allowing users to add a food list to other users list violates this policy.  Let's ensure each user creates his/her own list and can only edit the list created by him/herself, by removing the user id specificity in the food form.

![image](https://github.com/MayPyone/Recipe-app/assets/12745474/22d85259-bc75-481c-8985-ce9dff880a42)